### DISCUSSION TITLE
Deployment is ready but URL shows NOT_FOUND

### DISCUSSION THREAD

**Discussion by luismzk** [vercel/community#7405] [Help]
### Summary

Hello, I'm having an issue deploying a basic project on the Vercel platform. The logs look fine and everything should be working, but when I access the url either [here](https://teammatcher-eja1ymngn-luis-martinez-s-projects.vercel.app/) or [here](https://teammatcher.vercel.app/) I get a "NOT_FOUND" error. 
If I run the project with the Vercel CLI it runs fine but on the platform it doesn't work. 

Log output:
```
teammatcher@1.0.0 build
tsc && npm run copy-files && cd dist && ls
teammatcher@1.0.0 copy-files
cp -r ./src/views ./dist/ && cp -r ./src/public ./dist/
public
routes
routes.js
server.js
utilities.js
views
Build Completed in /vercel/output [7s]
Deploying outputs...
Injected preview comments middleware, you can disable this in your project settings.
Deployment completed
Uploading build cache [8.55 MB]...
Build cache uploaded: 338.041ms
```

Any tips? I'd like to move on to a more complex project but I'd like this one to work first 😅 

### Example

https://teammatcher.vercel.app/

### Steps to Reproduce

Open either of these links, it should return a NOT_FOUND error
https://teammatcher.vercel.app/
https://teammatcher-eja1ymngn-luis-martinez-s-projects.vercel.app/

**Comment by pawlean** [comment]
Hi @luismzk,

What does your [root directory settings](https://vercel.com/docs/deployments/configure-a-build#root-directory) look like? Might be worth looking at your deployment settings and seeing if it points to the correct directory.

Let us know how you get on!

[a comment by an automated account is not shown]
